.github/workflows/cicd.yml

```diff
@@ -73,7 +76,3 @@ jobs:
         env:
           IMAGE_TAG: ${{needs.build_and_publish.outputs.image_tag}}
           ENVIRONMENT: staging
-          ARM_CLIENT_ID: ${{ fromJSON(secrets.AZURE_CREDENTIALS).clientId }}
```

Comment:
This step runs the `cideploy` script, which calls the `deploy` script (which in turn sources or calls other scripts in `deployment/bin`). I think some of the commands in those scripts are still explicitly authenticating with SP auth and will need to be adjusted to not use a client secret.